variants/fri3d_2024_esp32s3/pins_arduino.h

```diff
@@ -0,0 +1,108 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+#include "soc/soc_caps.h"
+
+#define USB_VID 0x303a
+#define USB_PID 0x1001
+
+#define EXTERNAL_NUM_INTERRUPTS 46
+#define NUM_DIGITAL_PINS        48
+#define NUM_ANALOG_INPUTS       20
+
+static const uint8_t LED_BUILTIN = 21;
+#define BUILTIN_LED  LED_BUILTIN // backward compatibility
+#define LED_BUILTIN LED_BUILTIN
+#define RGB_BUILTIN LED_BUILTIN
+#define RGB_BRIGHTNESS 64
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<49)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 46)
```

Comment:
This part made been moved from pins_arduino.h file, and is defined globally per SoC.
Please remove it.
```suggestion
```